### ISSUE TITLE
[Update] Approve "Correcting Scene Titles"

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#26]
Needs official approval:
https://guidelines.stashdb.org/docs/scenes/scene-titles/#correcting-scene-titles

The reasoning behind this one isn't immediately obvious and so it also isn't a particularly well-known guideline. The idea here is that it is harder to find the scene elsewhere if the title has been "corrected" in some way. Consideration is also made for the addition of alternate titles so that scenes can track both original unaltered titles in addition to titles "corrected" according to an agreed-upon style guide for StashDB. Speaking of, we would also obviously need to agree on what that style guide should be before we would even know what kinds of corrections we'd want to make.

**Comment by DogmaDragon** [comment]
> We also do not alter the official studio’s formatting when they do specify a scene in their title.

Don't like this part. Modifying movie/scene number stylization doesn't alter scene/movie searchability but makes it look all over the place when there is no uniform standard. 

Just a few studio variations to give an idea of how inconsistent it is:
- `Uncensored Cumshots #04 Scene 7`
- `Sex and Corruption Episode 3 - Scene 5`
- `Ease His Ache Scene 3`
- `Student Bodies 7 Part 1: As Sweet As Apple Pie! Scene 1`
- `Teens Gone Black , Scene 4`
- `Girls In Gear Volume 04 Scene 5`
- `Lesbian Mentors Vol 03 Scene 2`

**Comment by PhuriousGeorge** [comment]
@DogmaDragon I agree with the random scene/part/season/episode/volume formatting frustration.  I'm standardizing on my end, but not certain if there's an appropriate path for standardization in a community DB.  _If_ we had the ability for "secondary titles", then we could hide the ugly formatting there that could also be found when searching and produce a more favorable consistency if so desired as the "display title"

We also have the Nubiles go-to of `S00E00` for all of their scenes.  I've seen about a 60/40 split in favor of removing the S/E information, so I've been adding those back as `Studio Codes` when I come across them.

**Comment by Valmox** [comment]
I'd _love_ to see this in the form of a feature which would take it beyond the remit of guidelines. If we were to allow this in general, I think it would be best to have a field for "Original Studio Title" and use the primary one as a "Clean Title". I suspect a great many of us put the effort into tidying titles (in our personal stash) to at least keep them consistent; I see no reason not to pursue this at a DB level. But, I wouldn't do it without being able to store the original title separately.

One thing that comes to mind is how often I struggle to find a scene on IAFD because they've altered a title too much.

If we did go forward with a clean title, I think the format hierarchy should be: [Series Name] [Scene ## ]: [Scene Name/Subtitle]

Example: "Yet Another Porn Scene 05: All Good Things..."

I'd also go with Title Case only.

**Comment by AdultSun** [comment]
Just linking an old issue I wrote regarding alternate titles in Stash-Box: https://github.com/stashapp/stash-box/issues/345

**Comment by BlokeCDP** [comment]
I'd remove the leading zero:
`Yet Another Porn Scene 5: All Good Things...`

From RenamerOnUpdate I've settled on:
`Title: (Act|Chapter|Episode|Part|Scene) \d: Subtitle`

Not sure if of any use.
